gcloud/storage/api.py

```diff
@@ -143,6 +135,9 @@ def create_bucket(bucket_name, project=None, connection=None):
 
     This implements "storage.buckets.insert".
 
+    If the bucket already exists, will raise
+    :class:`gcloud.exceptions.Conflict`.
+
```

Comment:
We essentially want to document the responses of the API methods / discovery resource methods, e.g. `storage.buckets.insert` here.

It's nowhere to be found in the discovery doc: https://www.googleapis.com/discovery/v1/apis/storage/v1/rest

Can we just leave as is and potentially file a bug to discuss with Travis?